test/component/uk/gov/hmrc/individualsemploymentsapi/controller/v2/SandboxEmploymentsControllerSpec.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package component.uk.gov.hmrc.individualsemploymentsapi.controller.v2
+
+import component.uk.gov.hmrc.individualsemploymentsapi.stubs.BaseSpec
+import play.api.libs.json.Json.parse
+import play.api.test.Helpers._
+import scalaj.http.{Http, HttpResponse}
+
+class SandboxEmploymentsControllerSpec extends BaseSpec {
+  private val employmentsScope = "read:individuals-employments"
+
+  feature("View individuals root (hateoas) entry point is accessible") {
+
+    scenario("missing match id") {
+      When("the root entry point to the API is invoked with a missing match id")
+      val response = invokeEndpoint(s"$serviceUrl/sandbox")
+
+      Then("the response status should be 400 (bad request)")
+      assertResponseIs(response, BAD_REQUEST, """
+          {
+             "code" : "INVALID_REQUEST",
+             "message" : "matchId is required"
+          }
+        """)
+    }
+
+    scenario("malformed match id") {
+      When("the root entry point to the API is invoked with a malformed match id")
+      val response = invokeEndpoint(s"$serviceUrl/sandbox?matchId=malformed-match-id-value")
+
+      Then("the response status should be 400 (bad request)")
+      assertResponseIs(
+        response,
+        BAD_REQUEST,
+        """
+          {
+             "code" : "INVALID_REQUEST",
+             "message" : "matchId format is invalid"
+          }
+        """
+      )
+    }
+
+    scenario("invalid match id") {
+
+      When("the root entry point to the API is invoked with an invalid match id")
+      val response = invokeEndpoint(s"$serviceUrl/sandbox?matchId=0a184ef3-fd75-4d4d-b6a3-f886cc39a366")
+
+      Then("the response status should be 404 (not found)")
+      assertResponseIs(
+        response,
+        INTERNAL_SERVER_ERROR,
+        "{\"statusCode\":500,\"message\":\"NOT_IMPLEMENTED\"}"
+      )
+    }
+
+    scenario("valid request to the sandbox implementation") {
+      When("I request the root entry point to the API")
+      val response = invokeEndpoint(s"$serviceUrl/sandbox?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430")
+
+      Then("The response status should be 200 (ok)")
```

Comment:
Test specifies 200 then we expect INTERNAL_SERVER_ERROR?

Please correct all occurrences.